br/pkg/lightning/config/config_test.go

```diff
@@ -955,3 +956,41 @@ func TestCheckAndAdjustForLocalBackend(t *testing.T) {
 	cfg.TikvImporter.SortedKVDir = base
 	require.NoError(t, cfg.CheckAndAdjustForLocalBackend())
 }
+
+func TestCreateSeveralConfigsWithDifferentFilters(t *testing.T) {
```

Comment:
please add a false case

Reply:
It has already included the false case.  If you change the implementation of `GetDefaultFilter()` by just returning `defaultFilter`, you'll find this test won't pass.

Reply:
yes, you are right. You can add what you said into this ut.